web/views/rule_view.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Cog.V1.RuleView do
+  use Cog.Web, :view
+
+  def render("rule.json", %{rule: rule}) do
+    ast = to_ast(rule)
+    %{id: rule.id,
+      command: ast.command,
+      rule: to_string(ast)}
+  end
+
+  def render("rule_expression.json", %{rule: rule}) do
+    rule |> to_ast |> to_string
+  end
```

Comment:
Good catch! I feel like I added it thinking I was going to use it but never did. I'll delete that.